packages/gatsby-source-nacelle/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "gatsby-source-nacelle",
```

Comment:
Thoughts on scoping this to `@nacelle`? It's redundant, I know... That's where all of our other React packages live though. 🤔  `@nacelle/gatsby-source-plugin`? Unsure if that's doable or even makes sense.

Reply:
Good catch! It should absolutely be scoped to `@nacelle`.